code/modules/clothing/head/jobs.dm

```diff
@@ -1,3 +1,5 @@
+/obj/item/clothing/head/

```

Comment:
No need to be sassy, I was genuinely asking a question. Is that ok there or should it be moved to here?
 https://github.com/tgstation/tgstation/blob/eace4eec148e8a366ee2fe6673f852b59a2e77b1/code/modules/clothing/clothing.dm#L187